cfme/tests/automate/test_git_import.py

```diff
@@ -449,3 +449,82 @@ def test_automate_git_import_without_master(appliance, request):
     )
     domain = repo.import_domain_from(branch="origin/testbranch")
     request.addfinalizer(domain.delete_if_exists)
+
+
+@pytest.fixture
+def setup_datastore(appliance):
+    """This fixture create setup of datastore similar to domain which we are going to import.
+       Note: Names of domain, namespace, klass, method are hardcoded as per their names in
+       datastore which we are going to import.
```

Comment:
I think on a different PR we discussed that it wasn't possible to fetch these values from the imported domain. In lieu of that, I think we should store this `imported_domain` information somewhere in a yaml to avoid duplication of the hardcoded strings. What do you think? I would be okay with a follow-up PR to accomplish this.

Reply:
Hey @john-dupuy 
- Actually, I am not getting, how it will avoid duplication by storing these values in yml. 
- So I modified it like, we have __init__ and I already have some variables there. So created one dict for these values and imported in test case.
- Please let me know, how you want me to modify or update done by me is fine or not?